platform/run_image_test.go

```diff
@@ -52,27 +52,53 @@ func testRunImage(t *testing.T, when spec.G, it spec.S) {
 			})
 
 			when("contains an image matching run image ref", func() {
-				inputs.RunImageRef = "some-run-image-from-run-toml"
+				inputs.RunImageRef = "some-run-image-from-run-toml-1"
```

Comment:
Added `-1` because by default we return the first image in run.toml and the test still passes